src/server/game/Instances/InstanceSaveMgr.cpp

```diff
@@ -789,5 +789,11 @@ void InstanceSaveMgr::UnbindAllFor(InstanceSave* save)
 {
     GuidList& pList = save->m_playerList;
     while (!pList.empty())
```

Comment:
This needs to be rewritten. For ref: https://github.com/azerothcore/azerothcore-wotlk/pull/11003/